web/src/app/administration/workbasket/details/workbasket-details.component.html

```diff
@@ -6,7 +6,7 @@
                 <a (click)="backClicked()">
```

Comment:
General comments.
 - I do not understand what is Callback information, imagine what should think a user, we need a more descriptive naming, also It looks like they are the same thing, just an object with attribute/value

- Think if you want to add reset functionality to the form(up to you)
- When saving a screen I would avoid to the user do more actions, thats why I create the global spinner service to block the ui, use this instead of local spinner for saving.
- After saving please show snack bar on the bottom for showing feedback to the user
- Personally I prefer to use a collapsible approach instead of using tabs for the same form (but again this is up to you.)
- Bootstrap(use btn-group class on the button group for bootstrap I will make your top buttons prettiest)
- Do not create a function for parsing from object to map. If you need to transform data form the REST API do it in the service which is retrieving the data usin a pipe and map

Reply:
reset functionality: new ticket

Reply:
collapsible approach: new ticket

Reply:
reset function: done
global spinner: done
snack bar: done
bootstrap buttons: done
parsing now done by rest-service: done